tests/Toolkit/VTest.php

```diff
@@ -68,6 +68,9 @@ public function testCallInvalidMethod()
 		V::fool('me');
 	}
 
+	/**
+	 * @covers ::__callStatic
```

Comment:
Not sure about those TBH. I'd expect a test for `__callStatic()` to test the logic inside the method (calling a validator that exists, one that exists but with different case, one that doesn't exist as well as passing arguments correctly to the validator). The actual validators are IMO more a case for `@coversNothing`.